x-pack/legacy/plugins/apm/public/components/shared/Links/DiscoverLinks/__test__/DiscoverLinks.integration.test.tsx

```diff
@@ -79,7 +72,7 @@ describe('DiscoverLinks', () => {
     } as Location);
 
     expect(href).toEqual(
-      `/basepath/app/kibana#/discover?_g=(refreshInterval:(pause:true,value:'0'),time:(from:now%2Fw,to:now))&_a=(index:apm-index-pattern-id,interval:auto,query:(language:kuery,query:'span.id:"test-span-id"'))`
+      `/basepath/app/kibana#/discover?_g=(refreshInterval:(pause:true,value:'0'),time:(from:now%2Fw,to:now))&_a=(index:'apm-*',interval:auto,query:(language:kuery,query:'span.id:"test-span-id"'))`
```

Comment:
Not sure why this was not a string before (with quotes). I think it's correct with quotes since this is how Discover formats it.